### ISSUE TITLE
Localisation of timepicker

### ISSUE THREAD

**Issue by falko100** [ripjar/material-datetime-picker#138]
Hi.

Is it possible to set the timepicker to 24:00 display?
And to change the language for days and months ?

Thanks in advance.

**Comment by jwhitfieldseed** [comment]
You can use moment locales to set the language for day and month, and change _some_ date formatting options - see https://github.com/ripjar/material-datetime-picker/issues/89#issuecomment-278151106.

I'm open to a PR to make the date formatting more customisable (by exposing moment format strings as options).

**Comment by falko100** [comment]
This changes output, but not  the language inside the datetimepicker right ?

**Comment by falko100** [comment]
Ahh I was using an outdated version :)

**Comment by falko100** [comment]
Is it possible to translate the OK and Cancel buttons?

**Comment by jwhitfieldseed** [comment]
Yes - #129 addresses this, but it needs a few changes before I can merge it.

**Comment by frutsy007** [comment]
How can i looking foto the perfect time
